dali/python/backend_impl.cc

```diff
@@ -771,6 +811,13 @@ void ExposeTensor(py::module &m) {
         },
       R"code(Passthrough, since the object is already an instance of `TensorCPU`.)code",
       py::return_value_policy::reference_internal)
+    .def("_make_copy", [](const Tensor<CPUBackend> &tl) {
```

Comment:
```suggestion
    .def("_make_copy", [](const Tensor<CPUBackend> &t) {
```
It is called `tl` but it is a Tensor